omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -17,6 +17,15 @@ def ownership
     {"owner" => owner, "group" => group}
   end
 
+  def apr1_password(password)
+    cmd = Mixlib::ShellOut.new("openssl passwd -apr1 '#{password}'")
```

Comment:
Should we use a full path to our embedded openssl install here?

@jaym Do we need to do anything here for FIPS?

Reply:
`OPENSSL_FIPS=1`

Reply:
i highly doubt that will work with fips. It might be safer to use `crypt()` instead. Not sure you can have a working system where `crypt()` doesn't work